### ISSUE TITLE
new bulk formula

### ISSUE THREAD

**Issue by hyunchul386** [NOAA-EMC/DATM-MOM6-CICE5#38]
- Current bulk formula in NEMS mediator has been used for fluxes over the sea ice.
For solving the SST increasing issue CFSR/GEFS fluxes with DATM-MOM6-CICE5, 
it is needed to use the bulk formula over the ocean as well as the ice.
- The issue is that the current bulk formula is using surface wind velocities, temperature, and humidity as inputs
and estimates 10 m height fields.
- DATM_input of CFSR/GEFS have the 10m wind velocity, 2mT, and 2mQ, which can be used in the bulk formula
- The new bulk formula module will use the DATM inputs directly, instead of the estimation, and be applied over the whole ocean and ice.

**Comment by JessicaMeixner-NOAA** [comment]
Out of curiosity, why are you using bulk over ice? Do you not have all the inputs the ice model is otherwise asking for?

**Comment by hyunchul386** [comment]
Indeed, NEMS mediator has used bulk formula over the ice as default, I guess that the ice cover is different CFSR/GEFS from the DATM-MOM6-CICE5, which can induce lot of issues over the mismatch, and bulk formula over the sea ice would be reduce the issue.

**Comment by DeniseWorthen** [comment]
CICE obtains the height-lowest values from the atm and uses essentially the same formulas used in the mediator_methods to calculate the fluxes over the ice: [ice_atmo.F90 #L72](https://github.com/NOAA-EMC/CICE5/blob/aac1c7e3141d31574b58a8ad18e3d2160bab99ff/lanl_cice/source/ice_atmo.F90#L72)

**Comment by hyunchul386** [comment]
I am checking the impacts of changes from the current bulk formula to the new bulk formula.
In CFSR forcing there is big difference 
![taux-45s](https://user-images.githubusercontent.com/14810515/87689545-5a0ae600-c756-11ea-8aee-d9c6aba4d73c.gif)

taux along 45S in 2011-10-01 from CFSR (green, 00Z), the new bulk form (black, 03Z) and the current bulk (red, 03Z).
It is shown the the new bulk formula is more realistic than the current one.

**Comment by hyunchul386** [comment]
In GEFS forcing, the current bulk formula and the new one show the similar pattern.

![taux_45s_gefs](https://user-images.githubusercontent.com/14810515/87699656-2a160f80-c763-11ea-8721-deb24c979b11.gif)
taux along 45S in 2011-10-01 from GEFS (green, 00Z), the new bulk form (black, 03Z) and the current bulk (red, 03Z).

**Comment by hyunchul386** [comment]
- As shown, the taux of the current bulk formula from CFSR forcing is severely underestimated, but the one of the current bulk from GEFS forcing is comparable with one of new bulk formula. 

- The new bulk formula uses 10mU/V for  calculation of wind stress. In the current bulk formula, the lowestlevel U/V are input from CFSR/GEFS, and 10mU/V are estimated from the lowestlevel U/V.

- the lowestlevel U/V of GEFS and CFSR is different
![UGRD_HYBLEV1](https://user-images.githubusercontent.com/14810515/87702419-6481ab80-c767-11ea-9a4e-55499f4b17a6.gif)
UGRD_HYBLEV1 along 45S in 2011-10-01 from GEFS (black) and CFSR (red)

- the new bulk formula consistently provides realistic taux, and the current bulk formula has strong dependency of the lowest level data type, like CFSR.
